### DISCUSSION TITLE
Disable 'UNLOCK' via parameter

### DISCUSSION THREAD

**Discussion by JamesMarcum** [dgreif/ring#639] [Ideas]
Love this utility (using it through https://github.com/tsightler/ring-mqtt-ha-addon ) but have noticed something that causes me concern.  The locks can be unlocked without requiring a code.  Once tied in to Alexa, that means people can just yell "Alexa, unlock front door" from outside and... they're in.  Can some change be made so if calling unlock lock it either fails or requires a verbal code?

Thanks for your consideration!

**Comment by dgreif** [comment]
@JamesMarcum I think that would be a customization in the ring-mqtt-ha-addon, rather than in this base library.  While it's possible to do something like that in the base api, it really comes down to how it is configured and handled in the mqtt side.
